frontend/integration-tests/tests/developer-catalog.scenario.ts

```diff
@@ -96,7 +98,42 @@ describe('Developer Catalog', () => {
     expect(srvClassFilterCount).toEqual(numCatalogTiles);
   });
 
-  it('creates a service instance and binding', async() => {
+  it('filters catalog tiles by \'Template\' type correctly', async() => {
+    await catalogPageView.catalogTiles;
+    await catalogPageView.clickFilterCheckbox('kind-template');
+    expect(catalogPageView.catalogTileFor('CakePHP + MySQL (Ephemeral)').isPresent()).toBe(true);
+    expect(catalogPageView.catalogTileFor('Python').isPresent()).toBe(false);
+  });
+
+  it('create a template instance', async() => {
+    await catalogPageView.catalogTiles;
+    await catalogPageView.clickFilterCheckbox('kind-template');
+    await crudView.isLoaded();
+    expect(catalogPageView.catalogTileFor('Dancer + MySQL (Ephemeral)').isDisplayed()).toBe(true);
+
+    await catalogPageView.catalogTileFor('Dancer + MySQL (Ephemeral)').click();
+    await catalogView.catalogDetailsLoaded();
+    await catalogView.createResourceButton.click();
+    await browser.wait(until.and(crudView.untilNoLoadersPresent, until.presenceOf(catalogView.instantiateTemplateFrom)));
+    await srvCatalogView.createButton.click();
+    await crudView.isLoaded();
+
+    const templateInstanceItem = $('.co-resource-item .co-m-resource-templateinstance');
+    const templateInstanceName = $('.co-resource-item__resource-name');
+    expect(templateInstanceItem.isDisplayed()).toBe(true);
+    expect(templateInstanceName.getText()).toContain('dancer-mysql-example-');
+    await $('.co-m-table-grid');
```

Comment:
This will likely break when we switch to PF4 tables. I'm not sure the the check is needed. It should render as soon as `crudView.isLoaded()` is true.

Reply:
In my local testing, it looks like without this wait, the following tests will fail. Let me double confirm and I will remove it if not actually needed